src/org/rascalmpl/interpreter/TestEvaluator.java

```diff
@@ -87,6 +87,10 @@ public static int readIntTag(AbstractFunction test, String key, int defaultVal)
 
     private void runTests(ModuleEnvironment env, List<AbstractFunction> tests) {
         testResultListener.start(env.getName(), tests.size());
+        if (tests.size() <= 0) {
+            return;
```

Comment:
should this also have a ` testResultListener.done();` call?

Reply:
definitely! good catch @DavyLandman . I removed the superfluous call to testResultListener.start instead. This will help a lot with the reporting in the IDE and github actions.